src/databricks/labs/ucx/workspace_access/clusters.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import InvalidParameterValue
+from databricks.sdk.service.compute import ClusterDetails, DataSecurityMode
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterAccess:
+    def __init__(self, installation: Installation, ws: WorkspaceClient, prompts: Prompts):
+        self._ws = ws
+        self._prompts = prompts
+        self._installation = installation
+
+    def list_cluster(self):
+        cluster_list = {}
+        clusters = self._ws.clusters.list()
+        for cluster in clusters:
+            if cluster.cluster_source is not None and cluster.cluster_source.name != "JOB":
+                cluster_list[cluster.cluster_name] = cluster.cluster_id
+        return cluster_list
+
+    def _get_cluster_id(self, cluster_id: str):
+        if cluster_id != "<ALL>":
+            return [x.strip() for x in cluster_id.split(",")]
+        cluster_list = []
+        clusters = self._ws.clusters.list()
+        for cluster in clusters:
+            if cluster.cluster_source is not None and cluster.cluster_source.name != "JOB":
+                cluster_list.append(cluster.cluster_id)
+        return cluster_list
+
+    def _get_access_mode(self, access_mode: str):
+        if access_mode in {"LEGACY_SINGLE_USER", "SINGLE_USER"}:
+            return DataSecurityMode.SINGLE_USER
+        return DataSecurityMode.USER_ISOLATION
+
+    def map_cluster_to_uc(self, cluster_id: str):
+
+        cluster_id_list = self._get_cluster_id(cluster_id)
+        spark_version = self._ws.clusters.select_spark_version(latest=True)
```

Comment:
Is this latest point release version or latest LTS version. We really want latest LTS version.

Reply:
It will be the latest LTS version.

Reply:
Then you need to use "long_term_support" argument or something like that